code-of-conduct.md

```diff
@@ -1 +1,76 @@
 # Triangle DevOps Code of Conduct
+
+Triangle DevOps is dedicated to providing a harassment-free community experience
+for everyone, regardless of gender, gender identity and expression, sexual
+orientation, disability, physical appearance, body size, race, age or religion.
+We do not tolerate harassment of community participants in any form.
+
+Community participants violating these rules may be sanctioned or expelled from
+the community at the discretion of the organizers.
+
+Harassment includes, but is not limited to:
+
+* Verbal comments that reinforce social structures of domination related to
+  gender, gender identity and expression, sexual orientation, disability,
+  physical appearance, body size, race, age, or religion.
+* Sexual images in public spaces
+* Deliberate intimidation, stalking, or following
+* Harassing photography or recording
+* Sustained disruption of talks or other events
+* Inappropriate physical contact
+* Unwelcome sexual attention
+* Advocating for, or encouraging, any of the above behaviour
+
+If you are being harassed, notice someone else is being harassed, or have other
+concerns, please contact an organizer immediately.
+
+## Enforcement
+
+Participants asked to stop any harassing behavior are expected to comply
+immediately. If a participant engages in harassing behaviour, the organisers
+retain the right to take any actions to keep events and online spaces as
+welcoming environments for all participants.
+
+This includes warning the offender or expulsion from meetings, events, and any
```

Comment:
This sentence is a bit awkward, it needs a smoother transition from warning to expulsion, or needs to be broken into two sentences. 

Actions organizers may take include warning the offender or expulsion of the offender form meetings, ....

^ might work a bit better. I think adding the subject of the offender again to explusion helps.